subsys/bluetooth/controller/ll_sw/ull_conn_iso_types.h

```diff
@@ -17,10 +21,12 @@ struct ll_conn_iso_stream {
 	struct ll_iso_datapath *datapath_in;
 	struct ll_iso_datapath *datapath_out;
 	uint32_t offset;        /* Offset of CIS from ACL event in us */
+	ll_iso_stream_released_cb_t released_cb; /* CIS release callback */
 	uint8_t  established;	/* 0 if CIS has not yet been established.
 				 * 1 if CIS has been established and host
 				 * notified.
 				 */
+	uint8_t teardown;	/* 1 if CIS teardown has been initiated */
```

Comment:
nitpick: This could be a bitfield together with `established`

Reply:
Ack